keyboards/sofle/keymaps/flare576/rules.mk

```diff
@@ -0,0 +1,6 @@
+MOUSEKEY_ENABLE = yes
+OLED_DRIVER_ENABLE = yes
+ENCODER_ENABLE = yes
+EXTRAKEY_ENABLE = yes
+TAP_DANCE_ENABLE = yes
+WPM_ENABLE = yes
```

Comment:
```suggestion
WPM_ENABLE = yes

SRC += graphics.c
```

Reply:
This actually exposes a code smell: `isJumping` and `showedJump` is defined in `graphics.c` but manipulated in `keymap.c` based on the status of the Space key.

One solution would be to update `enums.h` to be something like `shared.h` and define `isJumping` and `showedJump` in there, allowing both `graphics.c` and `keymap.c` to interact with them; is there a more common pattern than this I should follow?

Reply:
Awesome; thank you for the explanation! I've added the graphics.h and its import (and removed `static` from the function) - functions, looks good, and doesn't feel _icky_! 🤘